lib/trino-parquet/src/main/java/io/trino/parquet/writer/PrimitiveColumnWriter.java

```diff
@@ -276,7 +281,6 @@ private List<ParquetDataOutput> getDataStreams()
             dictionaryPagesWithEncoding.merge(new ParquetMetadataConverter().getEncoding(dictionaryPage.getEncoding()), 1, Integer::sum);
 
             primitiveValueWriter.resetDictionary();
-            updateBufferedBytes();
```

Comment:
not needed?

Reply:
at this point we're already flushing the row group, so updating buffered bytes won't have any effect